Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.md

```diff
@@ -28,7 +28,7 @@
 | PH2072  | Require local editorconfig                   | Disabled by default.  As of VS 16.3, [Analyzers can now see settings](https://developercommunity.visualstudio.com/content/problem/791119/editorconfig-has-stopped-working.html) of Editorconfig files in parent folders.|
 | PH2073  | Call extension method as instance            | If Foo is an extension method of MyClass, call it as `MyClass.Foo`. |
 | PH2074  | Avoid register in Dispose                    | Dispose methods should unregister rather than register.      |
-| PH2075  | Avoid AssemblyVersion change                 | Do not accidentally change AssemblyVersion, as it breaks runtime compatibility. Specify the intended `dotnet_code_quality.PH2075.assembly_version` in an .editorconfig specific to the project|
+| PH2075  | Avoid AssemblyVersion change                 | Do not accidentally change AssemblyVersion, as it breaks runtime compatibility. Specify the intended `dotnet_code_quality.PH2075.assembly_version` in an .editorconfig specific to the project. Because this analyzer is not associated with a .cs file, use a [.globalconfig](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/configuration-files#global-analyzerconfig) file when disabling this analyzer is desired, as [described here](https://github.com/dotnet/roslyn/issues/37876#issuecomment-635383233).|
```

Comment:
...is not associated with **any** file...

Reply:
Auto-merge was on.  Updated in new PR. https://github.com/philips-software/roslyn-analyzers/pull/143